gitfiti.py

```diff
@@ -98,7 +98,7 @@
 [1,0,1,0,1,0,1,0,0,0,1,1,1,0,0,1,0,1,0,1,0,1,1,1]]
 
 ASCII_TO_NUMBER = {
-  '_': 0,
+  '-': 0,
```

Comment:
Ha, well spotted!

That should explain why I had to replace 1s with 0s in e7e67834c873a68fd8ec2dcfbdcd1429660126e3!

Please rebase and adjust the `NUMBERS` two-dimensional list in the test so that it contains 1s instead of 0s again like the original `ONEUP` value (that should make the test pass again).